web/beacon-app/src/features/onboarding/components/steps/workspace/index.tsx

```diff
@@ -14,7 +14,8 @@ const WorkspaceStep = () => {
   const isInvited = isInvitedUser(profile);
   const { updateProfile, wasProfileUpdated, isUpdatingProfile, reset, error } = useUpdateProfile();
 
-  const hasError = error && error.response.status === 400; // this means the workspace is already taken by another user
+  // The backend returns a 400 if the workspace is not valid.
```

Comment:
A 409 is returned if the workspace exists. Users will currently see a toast message for that error since it isn't a 400.

Reply:
It would be great to update this Line too 
:https://github.com/rotationalio/ensign/blob/3803a7f043338aa8c1025b359182f2f5fc9d47cc/web/beacon-app/src/features/members/hooks/useUpdateMember.tsx#L20